src/apps/chifra/pkg/pinlib/download_test.go

```diff
@@ -13,13 +13,16 @@ import (
 
 var manifestJSONSource = `
 {
-  "fileName": "pin-manifest.json",
+  "version": "2",
+  "chain": "mainnet",
+  "chainId": 1,
   "indexFormat": "Qmart6XP9XjL43p72PGR93QKytbK8jWWcMguhFgxATTya2",
   "bloomFormat": "QmNhPk39DUFoEdhUmtGARqiFECUHeghyeryxZM9kyRxzHD",
   "commitHash": "f29699d3281e41cb011ddfbe50b7f01bfe5e3c53",
-  "prevHash": "QmP4i6ihnVrj8Tx7cTFw4aY6ungpaPYxDJEZ7Vg1RSNSdm",
-  "newBlockRange": "000000000-000350277",
-  "newPins": [
+  "names": "QmP4i6ihnVrj8Tx7cTFw4aY6ungpaPYxDJEZ7Vg1RSNSdm",
+  "timestamps": "QmcvjroTiE95LWeiP8HHq1YA3ysRchLuVx8HLQui8WcSBV",
```

Comment:
Same comment here, but in this case, the timestamp file will have been updated, so it's quite likely that a 'normal' user will never download this. There is only a binary file. The first time user, will want this file.

In both cases (names and timestamp) we will want to remove these files from our repo -- including back version if possible since these two files bloat the size of the repo hugely.